particle/particle/__main__.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import absolute_import
+from __future__ import print_function
+
+from .particle import Particle
+
+# This file requires Plumbum
+from plumbum import cli
+
+
+class DecayLanguageParticle(cli.Application):
```

Comment:
To be renamed, I guess ;-).

Reply:
Was planning to rewrite. Which I did. :)

Reply:
I've settled on:

```bash
python -m particle search 211
```
And, I've added, for your PDGID functionality:

```bash
python -m particle pdgid 211
```

Try it out! (And, I've renamed `pdgid.print_all()` to `print(pdgid.info())`)